pandas/tests/extension/base/missing.py

```diff
@@ -86,7 +86,7 @@ def test_fillna_limit_backfill(self, data_missing):
         expected = pd.Series(data_missing.take([1, 0, 1, 1, 1]))
         tm.assert_series_equal(result, expected)
 
-    def test_fillna_no_op_returns_copy(self, data):
+    def test_fillna_no_op_returns_copy(self, data, request):
```

Comment:
These `request` don't seem used? (and below)

Reply:
It's not needed in the base class, but test_numpy.py and test_sparse.py need to reference request, and in order to do so, they need the base class to declare it as a parameter.

Reply:
> and in order to do so, they need the base class to declare it as a parameter

Hmm really? I thought pytest was not picky about request being referenced in a subclass method?

Reply:
Ah...lemme call the superclass without request argument (which it doesn't need).

Reply:
That worked, and also cleaned up decimal/test_decimal as well.